### PR TITLE
fix(sidebar): fix broken links

### PR DESCRIPTION
**Pull request by fiji-flo**
### Description

fix broken links.

### CODE REVIEW

**Review by argl — APPROVED** [review]

File: files/sidebars/learnsidebar.yaml [hunk 1]
```diff
@@ -171,7 +171,7 @@ sidebar:
       - /Learn/JavaScript/Client-side_web_APIs/Video_and_audio_APIs
       - /Learn/JavaScript/Client-side_web_APIs/Client-side_storage
   - type: section
-    link: Forms
+    link: /Learn/Forms
```
Comment (IceBankMyElf2) [inline files/sidebars/learnsidebar.yaml:174]: And that's how you get it all over the place 😁

File: files/sidebars/learnsidebar.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ sidebar:
       - /Learn/Getting_started_with_the_web/Publishing_your_website
       - /Learn/Getting_started_with_the_web/How_the_Web_works
   - type: section
-    link: HTML
+    link: /Learn/HTML
```
Comment (IceBankMyElf2) [inline files/sidebars/learnsidebar.yaml:20]: This one

File: files/sidebars/learnsidebar.yaml [hunk 3]
```diff
@@ -209,7 +209,7 @@ sidebar:
       - /Learn/Accessibility/Mobile
       - /Learn/Accessibility/Accessibility_troubleshooting
   - type: section
-    link: Performance
+    link: /Learn/Performance
```
Comment (IceBankMyElf2) [inline files/sidebars/learnsidebar.yaml:212]: Yhup

Comment (IceBankMyElf2) [reply files/sidebars/learnsidebar.yaml:212]: - [ ] []()here

**Review by IceBankMyElf2 — COMMENTED** [review]
It's all about the time 😜 is going to be able to do it again tomorrow morning

**Pull request merged**